fcm_devices/settings.py

```
@@ -13,8 +13,14 @@ def __getattr__(self, attr):
 
 
 DEFAULTS = {
-    # api key from Firebase for sending push messages
-    "API_KEY": None,
+    # from firebase service account file, found at
+    # https://console.firebase.google.com/u/0/project/_/settings/serviceaccounts/adminsdk
+    "GOOGLE_SERVICE_ACCOUNT_INFO": None,
+    # this is redundant, as it can be inferred from credentials/account info,
+    # but alas is a required param for PyFCM at the moment.
+    # See https://github.com/olucurious/PyFCM/issues/357
```

Thoughts on removing this and just pulling it from the service account info, then passing it into PyFCM? Assuming that's trivial to do as you say

---

yep, we can do that! theoretically a cred file might not specify a project, so we probably would want to raise an `ImproperlyConfigured` error if the creds we parse are missing that

---

done